app/gui2/shared/yjsModel.ts

```diff
@@ -70,70 +70,81 @@ export class DistributedProject {
   }
 }
 
-export class DistributedModule {
-  doc: Y.Doc
+export class ModuleDoc {
```

Comment:
do we really need a class if it's just a POJO?

Reply:
(better question: do we need this class at all? if we ever want just the properties, we could do `{ ...theDistributedModule }`)

Reply:
It has a constructor that needs to be called exactly like that, so it's not really a POJO. I don't want this to be structurally typed. Also the Yjs types are not POJOs in the first place anyway.

Reply:
worth noting that in TS all classes are also structurally typed, unless they contain a protected/private/# property